### DISCUSSION TITLE
MAUI at #dotNETConf?

### DISCUSSION THREAD

**Discussion by Chiramisudo** [dotnet/maui#250] [General] [answered]
Are we going to hear any updates at #dotNETConf? The current Roadmap seems to hint at a possible Preview 1 reveal, and yet I'm not seeing evidence of some of the planned features actually having been completed? For example, the "Xamarin.Forms will become System.Maui" says "Committed" but I can't find where. The `git_branch` perhaps? It appears to have some partial work done.

At any rate, I have a new app idea that I'm very much looking forward to building on this when it becomes mature. 🤩

**Comment by GeraudFabien** [comment, marked as answer]
> Are we going to hear any updates

Highly doubt of that there may be some info but the developpement start next year. And decision start in december.

> At any rate, I have a new app idea that I'm very much looking forward to building on this when it becomes mature.

The first release will be with .Net6. Even if Maui will be a fork it will be a V1. If you thinks a V1 is "mature" then you may be able to use MAUI in november 2021. Otherwise you may wait until 2023.

You may want to use Xamarin or blazor (https://github.com/CloudNimble/ElectronNET.Blazor/)...
